lang/cs/Org.Apache.REEF.Demo/Task/Partition.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.Collections.Generic;
+
+namespace Org.Apache.REEF.Demo.Task
+{
+    public class Partition
+    {
+        public Partition(string id, IEnumerable<Block> blocks)
```

Comment:
Mm, I'm not quite sure.. I was considering this `Partition` class as a simple representation of an `IEnumerable` of `Block`s and nothing more, so I didn't include `IDataSetManager` as a member. A different class (task, maybe?) would use `IDataSetManager` to fetch these `Partitions`.